src/Stratis.Bitcoin.Features.Miner/BlockProvider.cs

```diff
@@ -30,6 +38,9 @@ public BlockTemplate BuildPosBlock(ChainedHeader chainTip, Script script)
         /// <inheritdoc/>
         public BlockTemplate BuildPowBlock(ChainedHeader chainTip, Script script)
         {
+            if (this.network.Consensus.IsProofOfStake)
```

Comment:
Ah you made the change here, I was planing to do it in the PowMiner.cs but I guess here is just as fine.